source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+Ordinarily, adding hosts in oVirt in the UI is tedious because you, as an administrator, have to set up the host 
+network on each new host.
+
+The Copy Host Networking feature makes adding new hosts easier because you can configure the network on a single host
+and then copy that configuration to any new host that has the same layout of network interfaces.
+
+You can use this feature to perform initial deployments and to update running environments. It is available as a button
+in the Administration Portal UI, and as a REST API method.
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Example flow
+
+After creating a cluster, you add several hosts (`Host0, Host1, Host2`). 
+You configure the network settings for `Host0`. The two other hosts have the same network 
+interface layout as `Host0`, so you copy the network configuration from `Host0` to `Host1` and `Host2`.
+
+In the Admin Portal, you go to the `Host1` and `Host2` network interfaces and click
+the `Copy Host Networking` button. In the popup window that appears,
+you select the host, `Host0`, from which to copy the configuration.
+
+Alternatively, with the REST API, you can use the `copynetworks` method for `Host1` and `Host2`. 
+This method takes a single parameter, the host ID from which to copy the configuration (`Host0`). 
+
+### Limitations
+
+* You can copy host configurations only within the same cluster. 
+
+* You cannot copy network configurations that contain static IP addresses: 
+This sets the boot protocol in the resulting network to `none`.
+
+* Copying a configuration to a host with the same interface names but different physical network connections will
+produce a bad configuration. Only copy configurations to hosts that have the same interface names and physical 
+network connections. 
+
+* The destination host has to have an equal or greater number of interfaces as the original host. Otherwise,
+the feature cannot determine what to do.
```

Comment:
"cannot determine"
What will be the outcome? Will it fail? Will it only do partial configuration?

Reply:
Done